packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -793,6 +794,36 @@ void main() {
     expect(tapOutsideCount, 3);
   });
 
+  testWidgetsWithLeakTracking('onTapOutside is not called upon tap outside when field is not focused', (WidgetTester tester) async {
```

Comment:
nit: I would add a regression test comment like:

`// Regression test for https://github.com/flutter/flutter/issues/134341.`